tests/test_recruiters.py

```diff
@@ -508,6 +508,23 @@ def test_submission_listener_enqueues_assignment_submitted_notification(
             mock.ANY, "AssignmentSubmitted", "some assignment ID", "some participant ID"
         ),
 
+    def test_clean_qualification_attributes(self, recruiter):
+        with open("datasets/example_prolific_details.json", "r") as f:
+            details = json.load(f)
+        cleaned_details = recruiter.clean_qualification_attributes(details)
+        assert details.keys() == cleaned_details.keys(), "Keys should be the same"
```

Comment:
I appreciate the effort made into making granular tests, but to be honest I think it would be clearest just to write something like

```py
assert recruiter.clean_qualification_attributes(details) == {
    # put the actual desired content here
}
```

This kind of test is actually often the easiest to debug when things go wrong.

Reply:
Yep, I agree. I implemented a variant of it. The full details is still too excessive. Comparing it against the `requirements` is sufficient I think.

Reply:
Cool